src/config/data.js

```diff
@@ -68,6 +68,7 @@ export const trainings = [
     discountPrice: 1450,
     nextDiscountPrice: 1579.17,
     priceGoesUpOn: 'Nov 2nd, 2018.',
+    countdownDate: new Date('november 3, 2018'),
```

Comment:
@ric9176 why do you want to add more stuff to the data? I think the countdownDate is a UI concern. Why don't we use priceGoesUpOn to calculate the countdown?

Reply:
Yes I agree :100: the reason I didn't initially is just that our priceGoesUpOn is just a string which is displayed, so it just needs a bit of refactoring so that it's a date object and then we generate the relevant string from that. TBH this is bc I did it very fast, I did have this refactor in mind tho :wink: